packages/cf-component-heading/src/HeadingCaptionTheme.js

```diff
@@ -0,0 +1,7 @@
+export default () => ({
+  color: '#434148',
```

Comment:
If it's a specific color for a specific component, it should define a variable in the global theme, and reference it locally.